CHANGES.md

```diff
@@ -1,3 +1,16 @@
+1.2.4
```

Comment:
Should be 1.3.0 i guess, changes are nontrivial (did you introduce any breaking compatibility changes?).

Reply:
Well, depends on what you assume is breaking compatibility. Function signatures didn't change. But if you have tool which parses logs then this tool will be broken.

Reply:
Ok.